plugins/woocommerce-admin/client/customize-store/assembler-hub/sidebar/global-styles/global-styles-variation-iframe/style.scss

```diff
@@ -73,10 +73,9 @@
 			}
 		}
 	}
+}
 
-	.block-editor-color-gradient-control__panel {
-		> .components-flex > .components-h-stack.components-v-stack {
-			display: none;
-		}
-	}
+// Hide "theme" and "default" sections
+.block-editor-color-gradient-control__panel > .components-flex > .components-h-stack.components-v-stack {
+	display: none;
```

Comment:
![image](https://github.com/woocommerce/woocommerce/assets/27843274/2f77d15e-24ba-4d95-ba0c-0939d9649e3b)

the css path structure looks different for the gradient tab, I suppose that is meant to be hidden as well?

Reply:
Ah, yea, good catch!

Reply:
Fixed in 615796ee2958b10bd86347fef80b1325123dc8ca